sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -91,10 +94,8 @@
 
   if False:  # pylint: disable=using-constant-test
     # This clause is interpreted by the compiler.
-    from cython import compiled as is_compiled
-  else:
-    is_compiled = False
-    fits_in_64_bits = lambda x: -(1 << 63) <= x <= (1 << 63) - 1
+    from cython import compiled as _is_compiled
+    is_compiled = _is_compiled
```

Comment:
I don't think this line would ever get executed. Instead, we may need to try the import and catch an import error. (We do want this bit to be set to True when it is compiled.)

Reply:
Ah. I see what you mean. Here are two possible approaches that avoids this problem:

- 3e7ccd9 depends on detecting pxd file is proceed and uses cimport cython explicitly
- eaa7cbd depends on the swaying behavior of 'import cython'. When processed with cython import cython is always treated as with an associated cimport. We probably want to avoid depending on this anomaly.